app-ios/Native/Sources/Feature/TimetableDetail/Components/SessionDescriptionView.swift

```diff
@@ -32,7 +32,7 @@ struct SessionDescriptionView: View {
                     isDescriptionExpanded = true
                     canBeExpanded = false
                 } label: {
-                    Text("続きを読む")
```

Comment:
The localized resources in SPM need the bundle to be set properly, so localization in Japanese failed to display.

<img width="456" height="972" alt="Image" src="https://github.com/user-attachments/assets/ef2f4c0d-4939-4ca4-a19f-e78a93c08da1" />

Reply:
Oops.
I forgot to specify bundle.
I'll fix it when I get home🙇‍♂️

Reply:
Fixed!
<img width="300" alt="simulator_screenshot_1AA0A190-E58F-4BD0-9BD8-879054D7F2B4" src="https://github.com/user-attachments/assets/288a899f-b886-4fa3-9727-b7fc47f07386" />
<img width="300" alt="simulator_screenshot_D2B0182B-3078-4E66-BC8A-AA7EAC43E283" src="https://github.com/user-attachments/assets/96dd1e8a-f765-4dd0-ae27-a9b750fafc0c" />